### ISSUE TITLE
Export first public dataset

### ISSUE THREAD

**Issue by jdpigeon** [NeuroTechX/neurodoro#25]
In order to help people contribute, we should be able to easily share all our CORVO data collected through the app.

There's a lot of opportunity for over engineering. To start, let's try using GCPs export function combined with a python script to clean and organize the data

**Comment by jdpigeon** [comment]
According to @cbattista, this should be as easy as importing it in a Jupyter Notebook

**Comment by cbattista** [comment]
Yeah - that's for users that are on the project.  if we want to make it
public there's an 'export to cloud storage' option on the BQ ui that we can
use.  Josh is gonna work on making that an automatic daily job.
-C
